recipes/r-sbl/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set version = '0.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sbl
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sbl_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sbl/sbl_{{ version }}.tar.gz
+  sha256: 7c5accfbe2e2e71c45c80610e8aafaa17ec436ce8cc9faac2ee54e1023d3bff3
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('sbl')"           # [not win]
+    - "\"%R%\" -e \"library('sbl')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=sbl
+  license: GPL-3.0-or-later
+  summary: Implements sparse Bayesian learning method for QTL mapping and genome-wide association
+    studies.
+  license_family: GPL3
+  license_file:
+    - 'GPL-3.txt'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - "Troelsmou"
```

Shouldn't need quotes.

```suggestion
    - Troelsmou
```

---

Also, as a first time contributor, please explicitly comment that you wish to be a maintainer. That is, you have the option to not be a maintainer if you'd like.